lib/xmlfilereader.php

```diff
@@ -104,7 +104,7 @@ public function getVerse($date = null)
 				),
 		);
 		
-		return apply_filters('herrnuterlosung_vers', $vers, $date);
+		return apply_filters('herrnhuterlosung_vers', $vers, $date);
```

Comment:
Again, this change would be good if not a BC break. So for backward compatibility we should (also) keep the old filter (and add a _doing_it_wrong-Notice).

Reply:
No threat to allow both.